.github/workflows/integration-tests.yml

```diff
@@ -0,0 +1,90 @@
+name: Integration Tests
+
+on:
+    workflow_dispatch:   
+jobs:
+    ui5-versions:
+        runs-on: ubuntu-latest
```

Comment:
should we also run it on windows? or will have a seperate job for it

Reply:
We can look into this in the future, but I'm not sure if it is worth it to run all the UI tests on windows. OS differences should be handled by non UI tests.